packages/block-editor/src/components/inserter/menu.js

```diff
@@ -215,6 +191,9 @@ function InserterMenu(
 		! delayedFilterValue &&
 		selectedMediaCategory;
 
+	// When the pattern panel is showing, we want to use zoom out mode
+	useZoomOut( showPatternPanel );
```

Comment:
Why have we moved this here, rather than keeping it inside `PatternCategoryPreviewPanel`?

Reply:
Before, the `PatternCategoryPreviewPanel` was mounted and then the patterns switched out inside of it. Now, each panel is its own component (via the tabs component). So, if we left it within `PatternCategoryPreviewPanel` it would rerun the hook each time you switched categories, toggling between zoom out and edit. By moving it up a level we can preserve the right mode.